docs/getting_started.rst

```diff
@@ -75,3 +75,39 @@ Subscribe to tag changes
 .. literalinclude:: ../examples/tag/subscribe_to_tag_changes.py
    :language: python
    :linenos:
+
+Data Frame API
+-------
+
+Overview
+~~~~~~~~
+
+The :class:`.DataFrameClient` class is the primary entry point of the Data Frame API.
+
+When constructing a :class:`.DataFrameClient`, you can pass an
+:class:`.HttpConfiguration` (like one retrieved from the
```

Comment:
parenthetical statements can be difficult to localize. if that's a concern, consider rewriting

Reply:
We don't currently localize these docs and don't have any immediate plans to so I think we can do without this change for now.